### ISSUE TITLE
Remove node version override

### ISSUE THREAD

**Issue by andreas-schilling** [rwinch/antora-gradle-plugin#12]
Currently the plugin sets a hard node version override when applied.
What is the exact reason for doing so?
I'd recommend to remove this override as it might be problematic if a script uses the Gradle Node plugin as well and desires different values. Even if one uses just the Antora Gradle plugin one can still apply the override by using the
```groovy
node {
    version = "whatever"
    download = true
}
```
configuration block, as the Gradle Node plugin becomes transitively applied.

**Comment by landsman** [comment]
Agree, I hit same issue with hardcoded version:

```
./gradlew antora
> Task :nodeSetup FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':nodeSetup'.
> Could not resolve all files for configuration ':detachedConfiguration3'.
   > Could not find org.nodejs:node:8.11.4.
     Searched in the following locations:
       - https://nodejs.org/dist/v8.11.4/ivy.xml
     Required by:
         project :

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/6.0.1/userguide/command_line_interface.html#sec:command_line_warnings

```

**Comment by rwinch** [comment]
Thanks. Would anyone be interested in submitting a Pull Request?

**Comment by rwinch** [comment]
This was fixed in 8e6596d9a05469897693dcb98576e16af44fbc02
